docs/topics/modules/glossary.adoc

```diff
@@ -83,7 +83,7 @@ pipeline:: Pipelines are a continuous delivery system that, at each step, tests
 restrictive license:: Licenses that are not commonly used in similar stacks or that do not work well with the stack’s representative license.
 // endterm
 
-//// term: 01e76137-ab89-4a3c-8765-48f54078154a, en_EN
+// term: 01e76137-ab89-4a3c-8765-48f54078154a, en_EN
```

Comment:
@mishaone This is intentional. The lines starting with `// term: ...` are there because the script that translates the glossary terms into JSON (`json/infotips.json`) uses it to read the UUID hash. So, in order to really comment out a term, the script needs to ignore the line, hence `////`.

Reply:
Ah, ok. I was changing it because it renders oddly (shows the entire line) in the UG. If you look at the end of the user guide on docs.openshit.io at the moment, it should show you what I mean.